presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergSessionProperties.java

```diff
@@ -48,6 +48,7 @@ public final class IcebergSessionProperties
 {
     private static final String COMPRESSION_CODEC = "compression_codec";
     private static final String PARQUET_MAX_READ_BLOCK_SIZE = "parquet_max_read_block_size";
+    private static final String PARQUET_WRITER_FORMAT_VERSION = "parquet_writer_format_version";
```

Comment:
I don't see this is being used?

Reply:
That is right. I see Iceberg is using hive session props to deal with parquet-related props.
I will remove it.

Reply:
Updated IcebergSessionProperties constructor to include this prop.